build.gradle

```diff
@@ -182,30 +182,30 @@ task starthere(type: Exec, dependsOn: [copyLocal, copyExtraLibs, copyTestLibs, c
 }
 
 task fastbuild(dependsOn: [
-    ':dbfit-java:core:check',
-    ':dbfit-java:db2:check',
-    ':dbfit-java:derby:check',
-    ':dbfit-java:hsqldb:check',
-    ':dbfit-java:mysql:check',
-    ':dbfit-java:netezza:check',
-    ':dbfit-java:oracle:check',
-    ':dbfit-java:postgres:check',
-    ':dbfit-java:sqlserver:check',
-    ':dbfit-java:teradata:check',
-    ':dbfit-java:derby:integrationTest',
-    ':dbfit-java:hsqldb:integrationTest'
+      ':dbfit-java:core:check'
```

Comment:
Ok, I don't mind that.

Any ideas on how to extend the build customisation using another (non version controlled) file (or something like environment variables)?